contents/blog/open-source-telemetry-ethical.md

```diff
@@ -0,0 +1,102 @@
+---
+date: 2020-09-21
+title: Should that open source project track you?
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+---
+
+Many open source projects now track their usage in some way, shape, or form.
+
+So much user tracking is horrible. It's usually buried deep in the legalese of a size 8 terms and conditions policy, and kept intentionally vague.
+
+*All tracking is bad*, many say. We disagree. Done right, it enables more and better software in the world. When it's applied to open source, it often enables more free software for anyone to use.
+
+That said, it is a topic that has to be approached with extreme caution. It is an ethical and business minefield - if you're working on open source, how do you decide what's ok?
+
+## An ethics framework
+
+### Duty-based ethics
+
+Morality is based on duty. That means that the measure of morality isn't the outcome of an act, but is rather based on the intention of the act itself.
+
+Stealing a packet of chips to save your child from starving would be considered wrong under duty-based ethics. Lying to save your friend from a murderer? Tut tut.
+
+### Utilitarian-based ethics
+
+Moral behavior means opting for the greatest common good. An act is moral if it increases the happiness or decreases the misery of the greatest number of people. The results are the measure of moral behavior, not the intent.
+
+Stealing those chips? Go for it.
+
+## So, is it ok for open source projects to track users?
+
+Open source projects are zero expectation - they may contain bugs, security risks, or have confusingly outdated documentation. Remember that time you lost hours on a project that just didn't work in the first place?
```

Comment:
I'd be a bit more formal here: "are zero expectation" is understood but not grammatically correct. "Based on the concept of" or something like that could help